src/redux/books/books.js

```diff
@@ -1,19 +1,26 @@
 import { createSlice } from '@reduxjs/toolkit';
+import { v4 as uuidv4 } from 'uuid';
 
 const booksSlice = createSlice({
   name: 'book',
   initialState: {
     booksList: [
       {
-        id: 2,
-        title: 'jame walker',
-        author: 'Ndorrh Oswald',
+        id: uuidv4(),
+        title: 'james walker',
+        author: 'Ndorrh',
 
       },
       {
-        id: 4,
+        id: uuidv4(),
         title: 'home coming',
-        author: 'Ndorrh Oswald',
+        author: 'Oswald',
+
+      },
+      {
+        id: uuidv4(),
+        title: 'Man of steel',
+        author: 'Peter',
 
       },
     ],
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/200757078-8298b2e4-bfeb-48a1-851c-05487b449dff.png)

- [ ] I am facing this error in the console window. Kindly fix it :blush:
- [ ] Also it would be nice if your remove all the hard coded books to make your initial state empty. This might fix the above error as well :+1:

Reply:
hi @HAMMAS-SALEEM  thanks for the review
Hi I think the error is there because I didn't add a key that is not actually require
The requirement of the project said we should add some books to the initial state to display in the UI